doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-dmn-based-proc.adoc

```diff
@@ -12,3 +12,13 @@ Create a test scenario template for DMN-based scenarios by following the procedu
 . Click *Ok* to create and open the test scenario in the test scenarios designer.
 +
 The template is automatically generated and you can modify it as per your needs.
++
+. To call a java method to a property cell during test scenarios execution:
```

Comment:
The explanation bellow is more about how to insert expression not how to run java method. Should we change it on "To be able to define java method for execution define an expression property"?

Reply:
Done